2022-02-28 12:45:13,372 INFO    MainThread:618916 [wandb_setup.py:_flush():71] setting env: {}
2022-02-28 12:45:13,372 INFO    MainThread:618916 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 12:45:13,372 INFO    MainThread:618916 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-28 12:45:13,373 INFO    MainThread:618916 [wandb_init.py:_log_setup():371] Logging user logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_124513-36jaubdv/logs/debug.log
2022-02-28 12:45:13,373 INFO    MainThread:618916 [wandb_init.py:_log_setup():372] Logging internal logs to /home/junshick/Workspace/pose_estimation/custom_pose/wandb/run-20220228_124513-36jaubdv/logs/debug-internal.log
2022-02-28 12:45:13,373 INFO    MainThread:618916 [wandb_init.py:init():404] calling init triggers
2022-02-28 12:45:13,373 INFO    MainThread:618916 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-28 12:45:13,373 INFO    MainThread:618916 [wandb_init.py:init():460] starting backend
2022-02-28 12:45:13,373 INFO    MainThread:618916 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-28 12:45:13,380 INFO    MainThread:618916 [backend.py:ensure_launched():216] starting backend process...
2022-02-28 12:45:13,384 INFO    MainThread:618916 [backend.py:ensure_launched():222] started backend process with pid: 618925
2022-02-28 12:45:13,385 INFO    MainThread:618916 [wandb_init.py:init():469] backend started and connected
2022-02-28 12:45:13,386 INFO    MainThread:618916 [wandb_init.py:init():533] updated telemetry
2022-02-28 12:45:13,387 INFO    MainThread:618916 [wandb_init.py:init():563] communicating current version
2022-02-28 12:45:13,947 INFO    MainThread:618916 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-28 12:45:13,947 INFO    MainThread:618916 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-28 12:45:14,254 INFO    MainThread:618916 [wandb_init.py:init():606] starting run threads in backend
2022-02-28 12:45:17,053 INFO    MainThread:618916 [wandb_run.py:_console_start():1810] atexit reg
2022-02-28 12:45:17,053 INFO    MainThread:618916 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-28 12:45:17,053 INFO    MainThread:618916 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-28 12:45:17,054 INFO    MainThread:618916 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-28 12:45:17,054 INFO    MainThread:618916 [wandb_init.py:init():633] run started, returning control to user process
2022-02-28 12:45:29,752 INFO    MainThread:618916 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-28 12:45:29,753 INFO    MainThread:618916 [wandb_run.py:_restore():1752] restore
2022-02-28 12:45:32,578 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 12:45:32,907 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 12:45:33,264 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 681
}

2022-02-28 12:45:33,365 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15571
}

2022-02-28 12:45:33,468 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15571
}

2022-02-28 12:45:33,569 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 681
  total_bytes: 15571
}

2022-02-28 12:45:33,671 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}

2022-02-28 12:45:33,773 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}

2022-02-28 12:45:33,876 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}

2022-02-28 12:45:33,978 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}

2022-02-28 12:45:34,081 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}

2022-02-28 12:45:34,480 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}

2022-02-28 12:45:35,077 INFO    MainThread:618916 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 15571
  total_bytes: 15571
}
local_info {
}

2022-02-28 12:45:36,145 INFO    MainThread:618916 [wandb_run.py:_append_history():2130] rendering history
2022-02-28 12:45:36,145 INFO    MainThread:618916 [wandb_run.py:_append_summary():2085] rendering summary
2022-02-28 12:45:36,145 INFO    MainThread:618916 [wandb_run.py:_append_files():2180] logging synced files
